test/JsonApiDotNetCore.MongoDb.Example.Tests/IntegrationTests/Filtering/FilterOperatorTests.cs

```diff
@@ -98,8 +98,7 @@ public async Task Cannot_filter_equality_on_two_attributes()
 
             responseDocument.Errors.Should().HaveCount(1);
             responseDocument.Errors[0].StatusCode.Should().Be(HttpStatusCode.BadRequest);
-            responseDocument.Errors[0].Title.Should().Be("MongoDB does not allow comparing two fields to each other, only constants.");
-            responseDocument.Errors[0].Detail.Should().Be("Unsupported filter: ({document}{SomeInt32} == {document}{OtherInt32}).");
+            responseDocument.Errors[0].Title.Should().Be("Comparing attributes against each other is not supported when using MongoDB.");
```

Comment:
Should also assert that .Detail is null. When that changes in the future, this breaks the test so we know about it.